src/pages/Main/SearchForm/SearchForm.js

```diff
@@ -1,23 +1,65 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
 import './SearchForm.scss';
 
 class SearchForm extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      searchValue: '',
+      matchResult: [],
+    };
+  }
+
+  searchText = e => {
+    this.setState({
+      searchValue: e.target.value,
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { searchValue } = this.state;
+    if (searchValue.length === 0) {
+      alert('검색어를 입력해 주세요.');
+    } else {
+      fetch(`http://10.58.2.176:8000/restaurants/search?search=${searchValue}`)
```

Comment:
```suggestion
      fetch(`${API}/restaurants/search?search=${searchValue}`)
```
src/config.js 경로에 api 주소 분리해주세요